### PR TITLE
Issue/2 hello react

### PR DESCRIPTION
**Pull request by hithwen**

### CODE REVIEW

File: 01 HelloReact/readme.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+export default HelloComponent;
+```
+
+- Wire up this component by using react-dom under _main.jsx_ (we have to rename this file
+  from js to jsx and replace the content).
+
+```javascript
+iimport React from 'react';
```
Comment (vicmator) [inline 01 HelloReact/readme.md]: import React from 'react';

File: 01 HelloReact/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+       inline: true, //Enable watch and live reload
+       host: 'localhost',
+       port: 8080,
+       stats: 'errors-only'
+  },
+
+  module: {
+    loaders: [
+			{
+				test: /\.jsx$/,
```
Comment (vicmator) [inline 01 HelloReact/webpack.config.js]: Fix that to put code inline better.

Comment (vicmator) [reply 01 HelloReact/webpack.config.js]: To explain better, "Code formatting (tabs)"

**Comment by brauliodiez** [comment]
@hithwen There are open points in this PR, do you have time to work on them or do you prefer another team mate to tackle on them? Thanks

File: 01 HelloReact/webpack.config.js (hunk 2 above)
Comment (hithwen) [reply 01 HelloReact/webpack.config.js]: Yes, I've configured atom now to display all whitespace characters

**Comment by hithwen** [comment]
Comments addressed

**Comment by brauliodiez** [comment]
Good stuff, going to merge, please @vicmator or @rocioghermoso update kanban and issue status, thanks.

**Pull request merged**